### PR TITLE
Update UG v1.1

### LINKED ISSUE #19: Update Draft UG from W6

### PR DESCRIPTION
**Pull request by kelvinvin**
fixes #19

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,178 +1,99 @@
...
+###### tags: `public`
 
-* Table of Contents
-{:toc}
+## v1.1
 
---------------------------------------------------------------------------------------------------------------------
+### User stories, User persona
 
-## Quick start
```
Comment (benclmnt) [inline docs/UserGuide.md]: Should we keep this?

**Comment by benclmnt** [comment]
I think it is better to keep the structure of AB3 UG for future reference

File: docs/UserGuide.md [hunk 2]
```diff
@@ -152,9 +143,9 @@ Format: `exit`
 
 AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
-### Archiving data files `[coming in v2.0]`
+### Adding and deleting gym sessions `[coming in v2.0]`
```
Comment (dhafinrazaq) [inline docs/UserGuide.md]: should this be mid-v1.2?

Comment (maguireong) [reply docs/UserGuide.md]: Maybe can update along the way?

Comment (dhafinrazaq) [reply docs/UserGuide.md]: okay agree

Comment (benclmnt) [reply docs/UserGuide.md]: maybe, coming in `mid-v1.2`? 😄

Comment (kelvinvin) [reply docs/UserGuide.md]: updated!

Comment (kelvinvin) [reply docs/UserGuide.md]: oops i just wrote v1.2. shud be fine?

**Review by benclmnt — APPROVED** [review]
Haven't look at the details, but overall looks good to merge

**Pull request merged**